electronics/electrical_impedance.py

```diff
@@ -0,0 +1,41 @@
+# https://en.wikipedia.org/wiki/Electrical_impedance
```

Comment:
Could you add a brief description here about the impedance?

Reply:
I have added the description for electrical impedance.